documentation/book/cluster-operator.adoc

```diff
@@ -1060,111 +1060,121 @@ They will not be added to the Docker image from a previous build.
 ----
 
 Note line 12, where the the `strimzi-cluster-operator` ServiceAccount is specified as the `serviceAccountName`.
 
-If the rack awareness feature is used, a Kafka init container will be started before the broker container in order to enable it.
-The application inside this container is best run using a ServiceAccount:
 
-[source,yaml,options="nowrap"]
-.Example ServiceAccount for the Kafka init container
-----
-include::examples/install/cluster-operator/04-service-account-kafka.yaml[]
-----
-
-The Cluster Operator will be in charge of configuring this ServiceAccount as `serviceAccountName` for the Kafka init container during cluster deployment.
-
-==== Defining a Role
+==== Defining ClusterRoles
 
-The Cluster Operator needs to operate using a Role that gives it access to the necessary resources.
-Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the Role.
-Cluster administrator rights are only needed for the creation of the Role.
-The Role itself does not contain any cluster administrator privileges, and the Cluster Operator will not run under the cluster admin account.
-The Role follows the "principle of least privilege" and contains only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect and Zookeeper clusters.
-The assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as StatefulSets, Deployments, Pods and ConfigMaps.
+The Cluster Operator needs to operate using some ClusterRoles that give it access to the necessary resources.
+Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the ClusterRoles.
+Cluster administrator rights are only needed for the creation of the ClusterRoles.
+The Cluster Operator will not run under the cluster admin account.
+The ClusterRoles follow the "principle of least privilege" and contain only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect and Zookeeper clusters.
+The first set of assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as StatefulSets, Deployments, Pods and ConfigMaps.
 
 [source,yaml,options="nowrap"]
 .Example Role for the Cluster Operator
 ----
-include::examples/install/cluster-operator/02-role.yaml[]
+include::examples/install/cluster-operator/02-ClusterRole-strimzi-cluster-operator-role.yaml[]
 ----
 
-When the Role cannot be created, the predefined ClusterRole named `edit` can be used instead.
-
-If the rack awareness feature is used, the started Kafka init container needs to operate using a ClusterRole that gives it access to the necessary resources.
+The `strimzi-kafka-broker` ClusterRole represents the access needed by the init container in Kafka pods that is used for the rack feature. As described in the <<delegated_privileges>> section, this role is also needed by the Cluster Operator itself in order to be able to delegate this access.
```

Comment:
Internal links should use the xref style instead of << >>  : http://strimzi.io/contributing/guide/#styling_links